src/_edx-bootstrap.scss

```diff
@@ -2,8 +2,10 @@
  * (those should go into SFE.scss). Because this file does not define styles, it is okay to import into every
  * individual component's SCSS file. */
 
+$base-rem-size: 0.625;
 @import '~@edx/edx-bootstrap/sass/edx/theme';
 @import '~bootstrap/scss/variables';
 @import '~bootstrap/scss/mixins';
 @import '~bootstrap/scss/utilities/spacing';
 @import '~bootstrap/scss/utilities/screenreaders';
+@import 'var-reset';
```

Comment:
Unfortunately we can't use the edx-boostrap legacy reset scss here because it also includes a lot of CSS definitions (the bootstrap reboot sass) that I don't want to include on every single component. This is just the variable reset portion of that file.